packages/distribution/addon/gql/queries/inquiry-incomplete.graphql

```diff
@@ -0,0 +1,18 @@
+query InquiryIncomplete($caseId: ID!) {
+  ready: allWorkItems(filter: [{ case: $caseId }, { status: READY }]) {
+    edges {
+      node {
+        id
+        status
+      }
+    }
+  }
+  suspended: allWorkItems(filter: [{ case: $caseId }, { status: SUSPENDED }]) {
+    edges {
+      node {
+        id
+        status
+      }
+    }
+  }
```

Comment:
You can do this in one graph by using inverted status filters. Also, you need to filter for inquiries, not all work items.

Reply:
Ah, that explains why I had different numbers in the mirage-graphql mock, I updated it to use a task filter and inverted status filters.